docs/code-splitting.md

```diff
@@ -0,0 +1,21 @@
+# Code Splitting
+
+## Overview
+
+The React app is configured to make use of [code splitting](https://webpack.js.org/guides/code-splitting) by default. When building the app for production, the code will be output in several chunks:
```

Comment:
To me, this documentation here seems to say that the Code Splitting is happening only for React app.
It should be made clear that the Code Splitting is supported for both

Reply:
I've only set it up for React. For Angular, there is route-level code splitting, which won't work with `@adobe/cq-spa-page-model-manager`. And component-level code splitting looks more complicated to set up, so I'm not sure if this should by in the archetype by default. See https://blog.angularindepth.com/dynamically-loading-components-with-angular-cli-92a3c69bcd28

Reply:
If a user wants code splitting and has updated their code to support it, they'll only have to change a few lines in `clientlib.config.js` so the library picks up the chunks and includes them in the `js` ClientLib folder.